google/cloud/pubsub/internal/streaming_subscription_batch_source_test.cc

```diff
@@ -1130,7 +1130,7 @@ TEST(StreamingSubscriptionBatchSourceTest, ExtendLeasesWithRetry) {
       std::make_shared<pubsub_testing::MockBatchCallback>();
   EXPECT_CALL(*mock_batch_callback, callback).Times(1);
   EXPECT_CALL(*mock_batch_callback, ModackStart).Times(2);
-  EXPECT_CALL(*mock_batch_callback, ModackEnd).Times(0);
+  EXPECT_CALL(*mock_batch_callback, ModackEnd).Times(2);
```

Comment:
I'd definitely prefer to have separate expectations for each ack id. Calling `ModackEnd("fake-001")` twice is not the same thing as calling `ModackEnd("fake-001")` once and `ModackEnd("fake-002")` once. But both would pass the expectations, as is.

```cc
EXPECT_CALL(*mock_batch_callback, ModackEnd("fake-001")); 
EXPECT_CALL(*mock_batch_callback, ModackEnd("fake-002"));
```

same goes for `ModackStart`

Reply:
Done